scripts/install_linux.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+# This script detects the current operating system architecture and installs the appropriate version of Ollama
+
+set -eu
+
+# Check for jq, systemd, and systemctl dependencies
+for cmd in "jq" "systemd" "systemctl"; do
+    if ! command -v $cmd > /dev/null 2>&1; then
+        echo "Error: $cmd is not installed, and this script requires it. Please install $cmd and try again."
+        exit 1
+    fi
+done
+
+# Determine the system architecture
+ARCH=$(uname -m)
```

Comment:
You might also want to check `uname -s` which will tell you if it's linux or macos